### ISSUE TITLE
Cannot reproduce the metrics from the Paper 

### ISSUE THREAD

**Issue by Theopetitjean** [ethnhe/FFB6D#62]
Hello,

I have some problems with the FFb6d code, in fact I have been trying for a few days to reproduce the results/metrics indicated on the paper.
I first ran the code with the pre-trained network provided by you, and got the desired results of course.
I then launched a local training on my machine (without any change brought to the code, I have only GitClone the FFb6d directory) and I obtain the following results:

kps_pth in get_kps: datasets/linemod/kps_orb9_fps/ape_8_kps.txt
loss_rgbd_seg 0.002694820675089778
loss_kp_of 0.4796774632306326
loss_ctr_of 0.037936181447335654
loss_all 0.5230032857826777
loss_target 0.5230032857826777
acc_rgbd 99.71056330771674
obj_id: 1 0.1 diameter: 0.010209865663
1 ape
***************add auc: 63.393883012269
***************adds auc: 90.00338269221461
***************add(-s) auc: 63.393883012269
***************add < 0.1 diameter: 2.1904761904761907
***************adds < 0.1 diameter: 85.90476190476191

Use time: 422.245965719223 s

Did I do something wrong during the git clone? Or did I forget a part? Because the metrics as we can see are below what it could/should be. Do you have an idea of what I can do to correct this problem? Or where this problem could come from.

Thanks in advance for your helps and your response

**Comment by qietingfengyin123** [comment]
I obtain the following results:
kps_pth in get_kps: datasets/linemod/kps_orb9_fps/ape_8_kps.txt
loss_rgbd_seg 0.003100496001896404
loss_kp_of 0.5232073128507251
loss_ctr_of 0.04493905678746246
loss_all 0.5743473608437039
loss_target 0.5743473608437039
acc_rgbd 99.66496068523044
obj_id:  1 0.1 diameter:  0.010209865663
1 ape
***************add auc:  56.269443099407404
***************adds auc:         81.65495292584156
***************add(-s) auc:      56.269443099407404
***************add < 0.1 diameter:       2.1904761904761907
***************adds < 0.1 diameter:      76.38095238095238

Use time:  113.4421911239624 s

Judging from the results of the last two lines, my accuracy is not high, may I ask what is the reason, have you solved it?
